frontend/src/containers/PublicGroup.js

```diff
@@ -79,7 +80,11 @@ export class PublicGroup extends Component {
             group={group}
             newUserId={newUser.id}
             closeDonationModal={this._closeDonationFlow.bind(this)} />
+          <section className='pt4'>
+            <RelatedGroups title={'Check out similar collectives'} groupList={group.related} />
```

Comment:
we need this in i18n, e.g. after you donate to PartidoDigital or LaPrimaire.org, you want to show text in spanish/french.